lighthouse-core/audits/metrics/largest-contentful-paint.js

```diff
@@ -36,12 +36,12 @@ class LargestContentfulPaint extends Audit {
    */
   static get defaultOptions() {
     return {
-      // 75th and 95th percentiles HTTPArchive -> median and PODR.
+      // 75th and 87th percentiles HTTPArchive -> median and p10 points.
       // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2020_02_01_mobile?pli=1
-      // Gives 2.5s roughly a score of 0.9. https://web.dev/lcp/#what-is-a-good-lcp-score
-      // see https://www.desmos.com/calculator/brcfwyox6x
-      scorePODR: 2000,
-      scoreMedian: 4000,
+      // Gives 2.5s a score of 0.9. https://web.dev/lcp/#what-is-a-good-lcp-score
+      // see https://www.desmos.com/calculator/1etesp32kt
+      p10: 2500,
```

Comment:
this one does approximate to a podr of 2251, not 2000, which is why the test samplejson score changed.

but 2500 is the https://web.dev/vitals/ "good" threshold for LCP, so this change is correct.

Reply:
you'll need to update sample json.

Reply:
> this one does approximate to a podr of 2251, not 2000, which is why the test samplejson score changed.
> 
> but 2500 is the https://web.dev/vitals/ "good" threshold for LCP, so this change is correct.

oops, I must have done this on purpose and then forgotten I did it. I was a little worried about the test failure :)

Reply:
> but 2500 is the https://web.dev/vitals/ "good" threshold for LCP, so this change is correct.

Oh I specifically raised this to Addy the other day about the confusion of reusing lighthouse design for metrics on different scoring scales. I thought all the vitals thresholds were for aggregate 75th percentile RUM data, are we intending on matching all the thresholds for slow 4g lab data?

Reply:
> Oh I specifically raised this to Addy the other day about the confusion of reusing lighthouse design for metrics on different scoring scales. I thought all the vitals thresholds were for aggregate 75th percentile RUM data, are we intending on matching all the thresholds for slow 4g lab data?

it was mostly a happy(-ish?) coincidence that the thresholds were close enough that it felt fine to use them (as fine as any other nearby rounded number, at least). But yeah, vitals are using aggregate data while we aren't, so even if we matched mobile scoring, if there's a desktop profile we wouldn't be matching there anyways.

Reply:
Yeah totally I don't have a problem making a minor shift here for a nice round number that happens to align, that sounds good 👍 

Just want to make sure it's not some sort of rule that our thresholds should be aligning with web vitals when we'll have multiple envs that *should* be different